app/css/common/button.css

```diff
@@ -0,0 +1,33 @@
+@value color-white, color-salem, darker-color-salem from './colors.css';
```

Comment:
Hahaha, postcss has a color plugin but I was hesitant to add it in because i wanted to keep it clean for first release. So i found the hex color for `color-salem` but `darker(color-salem, 10%)` which when evaluated to color-salem as well in `name that color` :(